oid4vci/pyproject.toml

```diff
@@ -31,7 +31,7 @@ pytest-ruff = "^0.3.2"
 aries-askar = { version = "~0.3.0" }
 indy-credx = { version = "~1.1.1" }
 indy-vdr = { version = "~0.4.1" }
-ursa-bbs-signatures = { version = "~1.0.1" }
```

Comment:
I've never known when this is required and when not? I think it's installed for every plugin, so we should probably change that.

Reply:
That may have been a stray change from me—that package causes issues on Apple Silicon, so I just removed it when I was testing, since this plugin doesn't use it. I do think you're right, in that it's installed for every plugin, so reverting that might not be a bad idea for consistency if nothing else

Reply:
Inclusion was making it difficult for @mepeltier to run the plugin since he's on M* Mac. I'll have a think about what the right way to handle this is :thinking:

Reply:
Looking over this again, I'm actually a bit confused what the integration dependency group represents. At least in this plugin, we have a separate poetry setup in the integration test directory that handles integration test dependencies.

Reply:
I think the idea was you have the required deps to run the agent and plugin in the outer level and then have additional ones in the integration test directory. I agree it should maybe be rethought a bit.

Reply:
> Inclusion was making it difficult for @mepeltier to run the plugin since he's on M* Mac. I'll have a think about what the right way to handle this is 🤔

Maybe just don't commit this change for now? Can leave it commented when developing on mac.

Reply:
Reverted